slices/openssl.yaml

```
@@ -23,6 +23,14 @@ slices:
       /usr/lib/ssl/openssl.cnf:
       /usr/lib/ssl/private:
 
+  # Include this slice and `ca-certificates_data` together to use the
+  # certificate bundle from `ca-certificates`.
+  # Missing this slice will cause applications using default certificates
+  # fail to verify local SSL certificates.
+  data:
```

```suggestion
  certs:
```

How about `certs` instead?

---

I would opt for `data`, as this aligns with the slice naming for `openssl` slice in noble.

---

Oh that's a shame. It seems like I was the one who introduced this slice. `certs` seems like a better name now. But anyway, yeah, since `data` is already there, let's go ahead with it.